ui/app.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const express = require('express')
+const path = require('path')
+const mongoose = require('mongoose')
+const bodyParser = require('body-parser')
+const session = require('express-session')
+const MongoStore = require('connect-mongo')(session)
+const helmet = require('helmet')
+const logger = require('./utils/logging/logger')
+const app = express()
+// const favicon = require('serve-favicon')
+
+// database connection ==================
+const dbUrl = ''
+mongoose.connect(dbUrl, (error) => {
+    if (error) {
+        logger.log('error', error)
+        throw new Error('Problem connecting to the database')
+    }
+    logger.log('info', 'Connected to the database')
+})
+
+// config ===============================
+app.set('views', path.join(__dirname, 'views'))
+app.set('view engine', 'pug')
+app.use(helmet())
+app.use(session({
+    secret: '',
+    resave: true,
+    saveUninitialized: false,
+    cookie: {
+        maxAge: 3600000 // expires in one hour
+    },
+    store: new MongoStore({
+        url: dbUrl
+    })
```

Comment:
This is a suggestion for a different PR, but all configuration information like secrets and DB urls + credentials should be held in a config file or pulled from environment variables. If you keep it in a file, make sure that file is ignored and setup is detailed in a README local to the app.